administrator/language/de-DE/com_modules.ini

```diff
@@ -26,6 +26,10 @@ COM_MODULES_CONFIGURATION="Module: Optionen"
 COM_MODULES_CUSTOM_OUTPUT="Benutzerdefinierte Ausgabe"
 COM_MODULES_CUSTOM_POSITION="Aktive Positionen"
 COM_MODULES_DESELECT="Abwählen"
+COM_MODULES_EMPTYSTATE_BUTTON_ADD="Modul erstellen"
+COM_MODULES_EMPTYSTATE_CONTENT="Module sind leichtgewichtige und flexible Erweiterungen, die beim Rendern von Seiten verwendet werden. Mit ihnen können kleine Funktionsblöcke in die Seite eingefügt werden."
+COM_MODULES_EMPTYSTATE_TITLE_ADMINISTRATOR="Es wurden noch keine Administrator-Module erstellt."
```

Comment:
```suggestion
COM_MODULES_EMPTYSTATE_TITLE_ADMINISTRATOR="Es wurden noch keine Administrator Module erstellt."
```

Reply:
Wir werden sowohl im Menü als auch in der Übersicht bisher "Administrator Module"